src/signals/incident-management/containers/IncidentDetail/components/Detail/index.js

```diff
@@ -1,68 +1,90 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Paragraph, themeColor, themeSpacing } from '@datapunt/asc-ui';
 
-import { incidentType, dataListType, attachmentsType } from 'shared/types';
+import { incidentType, attachmentsType } from 'shared/types';
 
 import { string2date, string2time } from 'shared/services/string-parser/string-parser';
 
-import './style.scss';
-
-import Highlight from '../Highlight';
 import Location from './components/Location';
 import Attachments from './components/Attachments';
 import ExtraProperties from './components/ExtraProperties';
 
-class Detail extends React.Component { // eslint-disable-line react/prefer-stateless-function
-  render() {
-    const {
-      incident, attachments, stadsdeelList, onShowLocation, onEditLocation, onShowAttachment,
-    } = this.props;
-
-    return (
-      <article className="detail">
-        <div className="detail__text" data-testid="detail-title">
-          {incident.text}
-        </div>
-
-        <dl>
-          <dt className="detail__definition">Overlast</dt>
-          <dd className="detail__value">
-            {string2date(incident.incident_date_start)} {string2time(incident.incident_date_start)}&nbsp;
-          </dd>
-
-          <Highlight
-            subscribeTo={incident.location}
-          >
-            <Location
-              incident={incident}
-              stadsdeelList={stadsdeelList}
-              onShowLocation={onShowLocation}
-              onEditLocation={onEditLocation}
-            />
-          </Highlight>
-
-          <Attachments
-            attachments={attachments}
-            onShowAttachment={onShowAttachment}
-          />
-
-          {incident.extra_properties && <ExtraProperties items={incident.extra_properties} />}
-
-          <dt className="detail__definition" data-testid="detail-email-definition">E-mail melder</dt>
-          <dd className="detail__value" data-testid="detail-email-value">{incident.reporter.email}</dd>
-
-          <dt className="detail__definition" data-testid="detail-phone-definition">Telefoon melder</dt>
-          <dd className="detail__value" data-testid="detail-phone-value">{incident.reporter.phone}</dd>
-        </dl>
-      </article>
-    );
+const DemiParagraph = styled(Paragraph)`
+  font-family: Avenir Next LT W01 Demi;
+  margin: ${themeSpacing(4)} 0;
+`;
+
+const DefinitionList = styled.dl`
+  margin: 0;
+  display: grid;
```

Comment:
Did we drop IE11 support? I thought the grid is not good supported on IE11.

Reply:
No, we didn't (officially). The use of CSS grid layout gracefully degrades, so it still looks okay.